src/thunderbots/software/ai/hl/stp/tactic/passer_tactic.cpp

```diff
@@ -38,6 +41,22 @@ double PasserTactic::calculateRobotCost(const Robot& robot, const World& world)
 std::unique_ptr<Intent> PasserTactic::calculateNextIntent(
     intent_coroutine::push_type& yield)
 {
+    MoveAction move_action = MoveAction(MoveAction::ROBOT_CLOSE_TO_DEST_THRESHOLD, true);
+    // Move to a position just behind the ball (in the direction of the pass)
+    // until it's time to perform the pass
+    while (curr_time < pass.startTime())
+    {
+        // We want to wait just behind where the pass is supposed to start, so that the
+        // ball is *almost* touching the kicker
+        Vector ball_offset =
+            Vector::createFromAngle(pass.passerOrientation())
+                .norm(DIST_TO_FRONT_OF_ROBOT_METERS + BALL_MAX_RADIUS_METERS);
```

Comment:
More of a note here (doesn't _have_ to be changed now), I think we should make `BALL_MAX_RADIUS_METERS` -> `2 * BALL_MAX_RADIUS_METERS`.

This is because it's very likely we will use this during free-kicks or set plays. In this case we have to be very careful about touching or moving the ball before we are truly ready to kick, otherwise the ball will be in-play and the other team may move to steal the ball or block it before we get the change to kick.

We saw on the field that the robot would sometimes touch the ball slightly and move it a bit. That will make the ball "in play" in a real game. By making the point slightly further from the ball we should be able to avoid touching it while still being really close.

Reply:
Great point, changed.